distribution/docs/src/main/jdocs/content/_plugins/tags-filter-plugin-contents.adoc

```diff
@@ -0,0 +1,23 @@
+:type: plugin
+:status: unpublished
```

Comment:
❓ Is it really unpublished if it's installed by default and required for interoperability?